Packs/XSOARmirroring/Integrations/XSOARmirroring/XSOARmirroring.py

```diff
@@ -400,9 +404,10 @@ def fetch_incidents(client: Client, max_results: int, last_run: Dict[str, Union[
             latest_created_time = incident_created_time
 
     # Save the next_run as a dict with the last_fetch key to be stored
-    next_run = {'last_fetch': (latest_created_time + timedelta(milliseconds=1))  # type: ignore[operator]
-                .strftime(XSOAR_DATE_FORMAT)}  # type: ignore[union-attr,operator]
-
+    next_run = {'last_fetch': (latest_created_time)  # type: ignore[operator]
```

Comment:
What will happen in a case when we support milliseconds (from 6 to 8)? We need to keep supporting this situation.

Reply:
We are supporting in milliseconds, I remove the timedelta(milliseconds=1) because I add the dedup, and it will prevent from duplication so we do not need to add time to the last fetch time.